app/controllers/display_orders_controller.rb

```diff
@@ -0,0 +1,20 @@
+class DisplayOrdersController < ApplicationController
+  before_action :authenticate_employee!
+
+  def index
+    case params[:filter]
+    when 'Delivered rewards'
+      render :index, locals: { orders: Order.where(employee: current_employee, status: :delivered)
+                                            .paginate(page: params[:page], per_page: 5)
+                                            .order(created_at: :desc) }
+    when 'Undelivered rewards'
+      render :index, locals: { orders: Order.where(employee: current_employee, status: :not_delivered)
```

Comment:
this might be a scope set on model

Reply:
Thanks a lot! According to your suggestion I have already moved filtering orders function into scope set on an order model.